setup.py

```diff
@@ -36,6 +44,7 @@
       maintainer='LabJack Corporation',
       maintainer_email='support@labjack.com',
       classifiers=CLASSIFIERS,
+      install_requires=["future"],
```

Comment:
This has been removed and the additional import in outputSinDAC.py was also removed.